src/styles/main.scss

```
@@ -4,3 +4,12 @@ body {
   color: $textPrimary;
   background-color: $actionBtn;
 }
+
+.container {
+  p {
+    color: blue;
```

Будь ласка спробуй використати SCSS зміні 
```suggestion
    color: $colorBlue;
```

А в файлі `_variables.scss` додай наступний код:
```
$colorBlue: blue;
```